app/utils/pages.tsx

```diff
@@ -10,26 +13,36 @@ interface IPage {
     path: string
 }
 
+export function getPathsToMap (){
+    const pathToLeagueData = path.join(process.cwd(), "app", "leagueConfiguration");
+    const pathsToMap = fs.readdirSync(pathToLeagueData);
+    return pathsToMap;
+}
+
+export function getConfigurationPage (filename:string){
+    return require(`../leagueConfiguration/${filename}`);
+}
+
+export function checkForSubpages (filename:string){
```

Comment:
nit: let's add a type
```suggestion
export function checkForSubpages (filename:string): boolean {
```